/*
 * motors_module.h
 *
 *  Created on: Nov 18, 2017
 *      Author: Admin
 */

#ifndef L5_APPLICATION_SOURCE_MOTORS_MODULE_H_
#define L5_APPLICATION_SOURCE_MOTORS_MODULE_H_


bool servo_motor_steer_angle (MOTOR_SIGNAL_t motor_steer);
bool dc_motor_drive_speed (MOTOR_SIGNAL_t motor_drive);


#endif /* L5_APPLICATION_SOURCE_MOTORS_MODULE_H_ */
